library/redis

```diff
@@ -2,14 +2,25 @@
 
 Maintainers: Tianon Gravi <admwiggin@gmail.com> (@tianon),
              Joseph Ferguson <yosifkit@gmail.com> (@yosifkit)
-GitRepo: https://github.com/docker-library/redis.git
+             David Maier <david.maier@redis.com> (@dmaier-redislabs)
```

Comment:
These are comma-separated, but realistically I think @yosifkit and I should probably be removed now -- we can't really be considered "maintainers" of a repository we don't have commit bits on, so even for the older content we're not really the maintainers anymore. :sweat_smile:

Reply:
If you adjust the content on line 59-63 of [generate-stackbrew-library.sh](https://github.com/redis/docker-library-redis/blob/master/generate-stackbrew-library.sh#L59-L63), then the output of that script should be contents of this `library/redis` file.

In other words:
```console
$ # from redis checkout
$ ./generate-stackbrew-library.sh > ../path-to-official-images-checkout/library/redis
```